### ISSUE TITLE
Published module 1.0.3 does not work with terraform 0.12

### ISSUE THREAD

**Issue by jayhding** [trussworks/terraform-aws-lambda-amiclean#8]
When I tried to migrate our codes to terraform 0.12 I found the module published in registry is not same as released version in GitHub.

Output from `terraform init`:
```
λ terraform init
Initializing modules...
Downloading trussworks/lambda-amiclean/aws 1.0.3 for ami-clean-lambda...
- ami-clean-lambda in .terraform/modules/ami-clean-lambda/trussworks-terraform-aws-lambda-amiclean-02a40dd
Downloading trussworks/lambda/aws 1.0.1 for ami-clean-lambda.amiclean_lambda...
- ami-clean-lambda.amiclean_lambda in .terraform/modules/ami-clean-lambda.amiclean_lambda/trussworks-terraform-aws-lambda-4817c0f
There are some problems with the configuration, described below.

The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.

Error: Unsupported argument


  on .terraform/modules/ami-clean-lambda.amiclean_lambda/trussworks-terraform-aws-lambda-4817c0f/main.tf line 45:
  45: locals = {

An argument named "locals" is not expected here. Did you mean to define a
block of type "locals"?
```

The content of `main.tf` is not same as in https://github.com/trussworks/terraform-aws-lambda-amiclean/archive/v1.0.3.zip

**Comment by rpdelaney** [comment]
Thank you for the report! We have added it to our backlog to address this.

**Comment by kilbergr** [comment]
Hi @jayhding  --we've released a new version (v2.0.0) that should be fully compatible. Please let us know if you have any further issues.
